parts/k8s/cloud-init/artifacts/cse_main.sh

```diff
@@ -63,7 +63,10 @@ else
     FULL_INSTALL_REQUIRED=true
 fi
 
-if [[ $OS != $COREOS_OS_NAME ]] && [[ $FULL_INSTALL_REQUIRED ]]; then
+if [[ $OS == $UBUNTU_OS_NAME ]] && [[ $FULL_INSTALL_REQUIRED ]]; then
```

Comment:
Now that this should be there via VHD we only do this in the full install scenario. Also changed `$OS != $COREOS_OS_NAME` to `$OS == $UBUNTU_OS_NAME` as that's a more expressive/accurate condition given the current level of OS support (i.e., we only support CoreOS and Ubuntu, so it's sort of unnecessarily obtuse to do the not condition IMO)

Reply:
I included this in #1043 since I'm already fixing that if statement

Reply:
thanks, I fixed the `[[ $FULL_INSTALL_REQUIRED ]];` thing in this PR as well